tools/iwyu/docker/Dockerfile

```diff
@@ -1,9 +1,10 @@
+# TODO: change "latest" to "develop"
```

Comment:
Does this need to be changed before merging?

Reply:
It needs to get changed *after* merging...